server/webdriver_tests/timeline_test.py

```diff
@@ -67,6 +67,8 @@ def test_charts_from_url_directly_and_uncheck_statvar(self):
         charts = self.driver.find_elements_by_class_name("card")
         # Assert there are two charts.
         self.assertEqual(len(charts), 2)
+        self.driver.set_window_size(width=1000,height=2000,windowHandle='current')
+        self.driver.save_screenshot("test_screenshots/test_charts_from_url_directly_and_uncheck_statvar.png")
```

Comment:
For the screenshot, can you undo the save_screenshot() in this file, create a new file "screenshot_test".py and do the screenshot in there.

Basically this file will be a central place to do all the screenshot. Please define a list of urls there and make screenshot for each url. A starting point could be:

/place?dcid=country/USA

/place?dcid=country/USA&topic=Demographics

/place?dcid=country/USA&topic=Health

/tools/timeline#&place=geoId/0606000,geoId/2511000,geoId/2603000,geoId/1777005,geoId/1225175,geoId/4815976&statsVar=Median_Age_Person

/tools/timeline#&place=geoId/0606000,geoId/2511000,geoId/2603000,geoId/1777005,geoId/1225175,geoId/4815976&statsVar=Count_CriminalActivities_ViolentCrime

/tools/timeline#place=country%2FUSA%2CgeoId%2F06085&pc=1&statsVar=CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedOrProbableCase

Reply:
Sure, SGTM

Reply:
Also can you name the saved file as "1_place_usa.png", "2_place_usa_demographics.png", ..., The name starts with the order we want them to appear in the screenshot page

Reply:
Good to know this!